baselines/fedprox/fedprox/main.py

```diff
@@ -53,7 +56,9 @@ def main(cfg: DictConfig) -> None:
     def get_on_fit_config():
         def fit_config_fn(server_round: int):
             # resolve and convert to python dict
-            fit_config = OmegaConf.to_container(cfg.fit_config, resolve=True)
+            fit_config: FitConfig = OmegaConf.to_container(  # type: ignore
+                cfg.fit_config, resolve=True
+            )
             fit_config["curr_round"] = server_round  # add round info
```

Comment:
Do we want to convert the `server_round` `int` to `float`, or rather change the type definition to include `int` as a possible value type?

Reply:
maybe in another pr another time